README.md

```diff
@@ -1,71 +1,54 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# SpaceX-Hub
+> In this project, I built a covid-19, data tracker that fetch the result of covid-19 in  Europe
+## Built with
+- HTML
+- CSS
+- React
+- Redux
+- JavaScript
+- VScode
+- Lots of love :heart:
+## Screenshot
+[Click here to see a screenshot](<img width="1440" alt="Screen Shot 2021-11-27 at 6 43 52 AM" src="https://user-images.githubusercontent.com/62937819/143659785-7746b52e-3a7b-495b-a550-16aeead6648c.png">
+)
+## Live Demo
+[Click here to see the project](https://dazzling-johnson-ead422.netlify.app/)
+
+
+## video demo
+[Click here to watch the video](https://www.loom.com/share/7b8d70d754984fd688df52b8bafc3ab8)
```

Comment:
- [ ] Please consider fulfilling one more requirement in the video presentation. I will quote it below:

---
> - you should also highlight some interesting piece of code or something you built that you are very proud of.
---